vita3k/gui/src/pkg_install_dialog.cpp

```diff
@@ -0,0 +1,123 @@
+// Vita3K emulator project
+// Copyright (C) 2020 Vita3K team
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+#include "private.h"
+
+#include <gui/functions.h>
+#include <host/functions.h>
+#include <host/pkg.h>
+#include <host/sce_types.h>
+#include <misc/cpp/imgui_stdlib.h>
+#include <util/log.h>
+
+#include <nfd.h>
+
+namespace gui {
+
+void draw_pkg_install_dialog(GuiState &gui, HostState &host) {
+    nfdresult_t result = NFD_CANCEL;
+
+    static nfdchar_t *pkg_path;
+    static bool draw_file_dialog = true;
+    bool is_entering_zrif = true;
+    static bool delete_pkg_file = false;
+    static const auto BUTTON_SIZE = ImVec2(60.f, 0.f);
+    bool pkg_success = false;
```

Comment:
Don't use `static` in methods! I'm scared, it can cause all sorts of weird stuff. I'd be happier with them as global variables.

If you need them to be persistent, keep the data in `GuiState`.